resolwe/flow/models/entity.py

```diff
@@ -26,3 +27,39 @@ class Meta(BaseCollection.Meta):
 
     #: indicate whether `descriptor` is completed (set by user)
     descriptor_completed = models.BooleanField(default=False)
+
+
+class RelationType(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+
+    ordered = models.BooleanField(default=False)
```

Comment:
Ordering for specific relation types can be implemented in packages where those types are defined (i.e. resolwe-bio). Normally there won't be lots of entities in single relation (<10), so ordering them on the client shouldn't be a problem.